### ISSUE TITLE
Code used to do analysis from blog?

### ISSUE THREAD

**Issue by ckelner** [MattsStats/Untappd-Potential#1]
Hi, really enjoyed http://mattsstatsblog.com/?p=466 -- I'm a pretty big untappd user myself and was curious if you were open to sharing the code used to analyze your data?

**Comment by MattsStats** [comment]
Yes! I'm going to throw up the R code I used when I have some free time
(hopefully soon)! Thanks for reading - really appreciate it
